molecule/kdc-with-replica/molecule.yml

```diff
@@ -0,0 +1,73 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+# Using geerlingguy's image that provides centos7 with ansible AND systemd
+platforms:
+  - name: "${MOLECULE_DISTRO:-centos7}-01.local"
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-centos7}-ansible:latest"
```

Comment:
I'ld prefer the new one from geerlingguy because we can avoid to (consume time to) build the custom image. Now pushing a commit to change the default scenario